src/monster.cpp

```diff
@@ -1591,7 +1591,7 @@ bool monster::is_immune_effect( const efftype_id &effect ) const
     }
 
     if( effect == effect_downed ) {
-        if( type->bodytype == "insect" || type->bodytype == "insect_flying" || type->bodytype == "spider" ||
+        if( type->bodytype == "insect" || type->bodytype == "insect flying" || type->bodytype == "spider" ||
```

Comment:
```suggestion
        if( type->bodytype == "insect" || type->bodytype == "flying insect" || type->bodytype == "spider" ||
```

I assume this is what you meant based on your summary? Otherwise, another oopsie.

Reply:
Oh for crying out loud, kids, don't not sleep and commit.